File: web2backend/web2_server/web2_server/Models/RentACar/RentACarCompany.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace web2_server.Models.RentACar
{
    public class RentACarCompany
    {
        [Key]
        public int Id { get; set; }

        public string Description { get; set; }
        public string CompanyName { get; set; }
        public string Adress { get; set; }

        public string PhoneNumber { get; set; }

        public ICollection<Car> Cars { get; set; }
    }
}

File: web2-ng-app/src/app/models/RentACarCompany.ts
export class RentACarCompany {
  Id: number;
  Description: string;
  CompanyName: string;
  Adress: string;
  PhoneNumber: string;

  constructor(
    Id: number,
    Description: string,
    CompanyName: string,
    Adress: string,
    PhoneNumber: string
  ) {
    this.Id = Id;
    this.Description = Description;
    this.CompanyName = CompanyName;
    this.Adress = Adress;
    this.PhoneNumber = PhoneNumber;
  }
}
